app/controllers/vimeo_controller.rb

```diff
@@ -1,3 +1,18 @@
 class VimeoController < ApplicationController
-  # SEARCH_URI = {METHOD}https://api.vimeo.com
+   SEARCH_URI = "https://api.vimeo.com/users?per_page=30&query="
+   TOKEN_HEADER = {  
+                    "Accept" => "application/vnd.vimeo.*+json;version=3.2",
+                    "Authorization" => "bearer #{ENV["VIMEO_ACCESS_TOKEN"]}" 
+                  }
+   
+   
+  def results
+    user = params[:query]
```

Comment:
can this variable be renamed to query or search_term? (something more descriptive of what the variable is doing?)